src/wled2.html

```diff
@@ -5,17 +5,28 @@
     defaults: {
       address: { value: "", required: true },
       brightness: { value: 128 },
+      enableBrightness: { value: true },
       delay: { value: 0, validate: RED.validators.number() },
+      enableDelay: { value: true },
       color1: { value: "#FFFFFF" },
+      enableColor1: { value: true },
       color2: { value: "#FFFFFF" },
+      enableColor2: { value: true },
       color3: { value: "#FFFFFF" },
+      enableColor3: { value: true },
       effect: { value: 0 },
+      enableEffect: { value: true },
       effectIntensity: { value: 128 },
+      enableEffectIntensity: { value: true },
       effectSpeed: { value: 128 },
+      enableEffectSpeed: { value: true },
       name: { value: "" },
       palette: { value: "0" },
-      preset: { value: "" },

```

Comment:
What does this mean for users upgrading from a prior version? If the stored value is the old empty string and they upgrade to this version what will happen? Will it correctly have no preset applied?

Reply:
If "" is sent through the code, it'll get converted to 0, and then fail on `if(0)` to set/send the preset. WLED, when setting a preset ID to 0, ignores the 0 and sets the preset ID to the next available number, so in this case, it should appropriately have no preset applied.